src/components/AuthButton.jsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import Button from "../views/Button/index";
+import { Logout } from "../redux/actions/loginAction";
+
+class AuthButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onLogout = this.onLogout.bind(this);
+  }
+
+  onLogout(event) {
+    const { dispatch } = this.props;
+    event.preventDefault();
+    dispatch(Logout());
+  }
+
+  render() {
+    const { isAuthenticated } = this.props;
+
+    return (
+      <div>
+        {isAuthenticated ? (
```

Comment:
Ternary operator arrangement according to Airbnb, can't find a link currently:

`isAuthenticated ? blahblah : blahblah` works for short ternaries

isAuthenticated 
  ?  blahblah
  :   blahblah
For longer ternaries like the one you have now

Reply:
@kimotho-njoki the auto formatting is formatting it to be the way it is currently. I do not know how to bypass it because It is raising an error if I arrange the ternary differently.